ci/2.7.pylintrc

```diff
@@ -1,113 +1,38 @@
+    too-many-return-statements, # (R0911): *Too many return statements (%s/%s)*
+    too-many-statements, # (R0915): *Too many statements (%s/%s)*
+    useless-object-inheritance, # (R0205): *Class %r inherits from object, can be safely removed from bases in python3*
+    useless-return, # (R1711): *Useless return at end of function or method*
+    bad-continuation, # Buggy, **REMOVED in pylint-2.6.0**
     bad-option-value, # pylint does not recognize the error code/symbol (needed to supress breaking changes across pylint versions)
     bad-whitespace, # Used when a wrong number of spaces is used around an operator, bracket or block opener.
     broad-except, # Used when an except catches a too general exception, possibly burying unrelated errors.
-    consider-iterating-dictionary, # Emitted when the keys of a dictionary are iterated through the .keys() method.
-    consider-using-dict-comprehension, # Although there is nothing syntactically wrong with this code, it is hard to read and can be simplified to a dict comprehension. (3+ only)
-    consider-using-set-comprehension, # Although there is nothing syntactically wrong with this code, it is hard to read and can be simplified to a set comprehension. (3+ only)
     deprecated-lambda, # Used when a lambda is the first argument to “map” or “filter”. It could be clearer as a list comprehension or generator expression. (2.7 only)
-    empty-docstring, # Used when a module, function, class or method has an empty docstring
-    line-too-long, # Used when a line is longer than a given number of characters.
-    misplaced-comparison-constant, # Used when the constant is placed on the left side of a comparison. 
     missing-docstring, # Used when a module, function, class or method has no docstring
-    missing-final-newline, # Used when the last line in a file is missing a newline.
-    multiple-statements, # Used when more than on statement are found on the same line.
-    no-self-use, # Used when a method doesn’t use its bound instance, and so could be written as a function.
     old-style-class, # Used when a class is defined that does not inherit from another class and does not inherit explicitly from “object”. (2.7 only)
-    superfluous-parens, # Used when a single item in parentheses follows an if, for, or other keyword.
-    too-many-statements, # Used when a function or method has too many statements. You should then split it in smaller functions / methods.
-    trailing-newlines, # Used when there are trailing blank lines in a file.
-    trailing-whitespace, # Used when there is whitespace between the end of a line and the newline.
-    useless-object-inheritance, # Used when a class inherit from object, which under python3 is implicit (3+ only)
-    duplicate-code, # Indicates that a set of similar lines has been detected among multiple file. TODO: this is on our "fix" list, and is thus left last.
     fixme, # Used when a warning note as FIXME or TODO is detected
```

Comment:
NIT: trailing comma on this line. There might be some issues with parsing the rcfile here if we ever add another section after this one.

Reply:
Yes, I noticed that extra comma and decided to leave it alone since it may be handy if adding more suppressions.